cmd/kubeadm/app/apis/output/v1alpha1/zz_generated.conversion.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
hm, i though this is now a requirement from go 1.17.

WRT kubeadm, could you please also update the kubeadm internal FG to GA:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/features/features.go#L42

i can add an AI to remove it in a following release.

Reply:
i will take care of kubeadm. you will have to review carefully since i rarely touch kubeadm code. 

As for the auto-gen stuff please ignore for now. my changes shouldn't create generated items, I am guessing i need a rebase. I will check it later

Reply:
i think for kubeadm 1.23 just flipping the FG to GA on this line should suffice: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/features/features.go#L42 

a slightly more complicated task would be the FG cleanup which would be scheduled for 1.24+, i guess.
can also be done async with help from contributors.

Reply:
So - do you want me to address the changes .. or?

Reply:
The agreement is to edit the feature file and leave the rest of kubeadm changes to kubeadm folks.